### PR TITLE
we should verify now

### PR DESCRIPTION
**Pull request by elimelt**
Author: verify

### CODE REVIEW

File: src/components/DiscordVerification.tsx [hunk 1]
```diff
@@ -1,78 +1,385 @@
...
+              Checking status for {MAX_ATTEMPTS * 5} seconds
+            </AlertDescription>
+          </Box>
+        </Alert>
+      ),
+    });
+  };
+
+  const copyCommand = () => {
+    navigator.clipboard.writeText(`/verify ${username}`);
```
Comment (ShawnCollinge) [inline src/components/DiscordVerification.tsx:169]: I don't think you can just paste it, also /verify opens a modal in discord IIRC

File: src/services/member.ts [hunk 2]
```diff
@@ -80,3 +80,14 @@ export async function updateMemberProfile(
 
   return deserializeMember(res.data);
 }
+
+// TODO: please god give me an endpoint for this
+export async function isCurrentMemberVerified(): Promise<boolean> {
```
Comment (ShawnCollinge) [inline src/services/member.ts:85]: You can just poll the member and if it's part of the group it's fine no?

**Review by ShawnCollinge — APPROVED** [review]
LGTM, should probably also add on the backend to make sure they are verified before accessing stuff.

File: src/services/member.ts (hunk 2 above)
Comment (elimelt) [reply src/services/member.ts:85]: yeah that's what this is doing rn, just feels silly to poll large object for 1 bit of data

File: src/components/DiscordVerification.tsx (hunk 1 above)
Comment (elimelt) [reply src/components/DiscordVerification.tsx:169]: ah rats you're right

Comment (elimelt) [reply src/components/DiscordVerification.tsx:169]: pasting just /verify and pushing enter makes the command selector pop up so I'll leave the copy in

**Pull request merged**